larreco/TrackFinder/PMAlgTrackMaker_module_continued.h

```diff
@@ -0,0 +1,1039 @@
+void PMAlgTrackMaker::produce(art::Event& evt)
+{
+
+	fDetProp = lar::providerFrom<detinfo::DetectorPropertiesService>();
+
+	reset(evt); // set default values, clear containers at the beginning of each event
+
+	pma::TrkCandidateColl result;
+
+	std::unique_ptr< std::vector< recob::Track > > tracks(new std::vector< recob::Track >);
```

Comment:
Consider replacing
  std::unique_ptr<std::vector<recob::Track>> tracks(new std::vector<recob::Track>);
with
  auto tracks = std::make_unique<std::vector<recob::Track>>();
and similarly for all other unique_ptr instances below.